### DISCUSSION TITLE
How to control the proportion of the text?

### DISCUSSION THREAD

**Discussion by lstgithub578** [tarikjabiri/dxf#117] [Q&A] [answered]
I’m writing GeoJSON to DXF and I want to write the text in the small green box, but the text is too large. What should I do?

**Comment by tarikjabiri** [comment, marked as answer]
Hi, you can control the text height, but its hard to know the text dimensions unfortunately. The best way is to use mtext entity. But it will be possible in the next version.

Also please paste images directly in the markdown editor. Do not provide external links.

**Reply by lstgithub578** [reply]
thanks!
